test/server/services/questions.js

```diff
@@ -0,0 +1,142 @@
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import * as chai from 'chai';
+import { test as testconfig } from '../../../knexfile';
+import Question from '../../../server/db/services/questions';
+import Model from '../../../server/db/models/questions';
+
+const knex = knexModule(testconfig);
+const knexdb = bookshelfModule(knex);
```

Comment:
this object should probably be called bookshelf. I know Duy used this pattern but its an artifact of a time when there was an extemporaneous import of bookshelf.